### PR TITLE
Add tutorial images and config 

### PR DESCRIPTION
**Pull request by lucas-tortora**
# Description of change

* Added images for all tutorials.
* Added config for tutorials page.
* Fixed sidebar.

## Type of change

- Documentation Fix

## How the change has been tested

Wiki was built locally.

## Change checklist

- [x] My code follows the contribution guidelines for this project
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

File: documentation/docusaurus.config.js [hunk 1]
```diff
@@ -28,7 +28,37 @@ module.exports = {
...
+                tags: ['text', 'getting-started', 'client', 'shimmer', 'js'],
+            },
+        ],
+        [
+            '@iota-wiki/plugin-tutorial',
+            {
+                title: 'Stardust for iota.js: Alias Transactions',
+                description:'Learn how to create a new Alias Transaction and update the Alias\'s state with iota.js.',
+                preview: '/img/tutorials/bg-alias-transactions.png',
+                route: '/shimmer/iotajs/tutorials/value-transactions/introduction',
```
Comment (jmcanterafonseca-iota) [inline documentation/docusaurus.config.js]: is this correct?

File: documentation/docusaurus.config.js [hunk 2]
```diff
@@ -28,7 +28,37 @@ module.exports = {
...
+                tags: ['text', 'getting-started', 'client', 'shimmer', 'js'],
+            },
+        ],
+        [
+            '@iota-wiki/plugin-tutorial',
+            {
+                title: 'Stardust for iota.js: Native Tokens',
+                description:'Learn how to mint, send and melt Native tokens with iota.js.',
+                preview: '/img/tutorials/bg-native-tokens.png',
+                route: '/shimmer/iotajs/tutorials/value-transactions/introduction',
```
Comment (jmcanterafonseca-iota) [inline documentation/docusaurus.config.js]: is this correct?

File: documentation/docusaurus.config.js [hunk 3]
```diff
@@ -28,7 +28,37 @@ module.exports = {
...
+                tags: ['text', 'getting-started', 'client', 'shimmer', 'js'],
+            },
+        ],
+        [
+            '@iota-wiki/plugin-tutorial',
+            {
+                title: 'Stardust for iota.js: NFTs and Unlock Conditions',
+                description:'Learn how to mint, send and claim NFTs using custom unlock conditions with iota.js.',
+                preview: '/img/tutorials/bg-nfts.png',
                 route: '/shimmer/iotajs/tutorials/value-transactions/introduction',
```
Comment (jmcanterafonseca-iota) [inline documentation/docusaurus.config.js]: is this correct?

**Comment by Dr-Electron** [comment]
I can't get the images to work 🤔 
Maybe @jlvandenhout can help?

File: documentation/docusaurus.config.js (hunk 2 above)
Comment (lucas-tortora) [reply documentation/docusaurus.config.js]: nope, it seems my last push was rejected.  Fixed :)

File: documentation/docusaurus.config.js (hunk 3 above)
Comment (lucas-tortora) [reply documentation/docusaurus.config.js]: nope, it seems my last push was rejected.  Fixed :)

File: documentation/docusaurus.config.js (hunk 1 above)
Comment (lucas-tortora) [reply documentation/docusaurus.config.js]: nope, it seems my last push was rejected.  Fixed :)

**Pull request merged**